django/contrib/admin/checks.py

```diff
@@ -987,9 +988,14 @@ def _check_actions_uniqueness(self, obj):
         """Check that every action has a unique __name__."""
         names = [name for _, name, _ in obj._get_base_actions()]
         if len(names) != len(set(names)):
+            for item, count in collections.Counter(names).items():
+                if count > 1:
+                    duplicate = str(item)
+                    break
+
             return [checks.Error(
-                '__name__ attributes of actions defined in %s must be '
-                'unique.' % obj.__class__,
+                '__name__ attribute of action "%s" defined in %s must be '
```

Comment:
I don't think putting the name after "action" here makes sense. The issue is that there are _two_ actions with the same name.

Reply:
Cool. I'll change it to something similar to your example `__name__ attribute of actions defined in %s must be unique. Duplicated names: %s`